core/payment/src/cli.rs

```diff
@@ -56,17 +68,19 @@ pub enum PaymentCli {
         #[structopt(subcommand)]
```

Comment:
I'm not sure, but seems that it is the only `address` within payment CLI. Shouldn't it be renamed to `account` to unify CLI?

Reply:
This is node ID, not payment account address. I'm not sure what to do with it. I don't even know if anyone is using this command.